app/templates/src/static/css/cf-theme-overrides.less

```diff
@@ -176,9 +176,12 @@
 // Color variables
 
 // .block
+@block__bg:                     @gray-10;
+@block__border:                 @gray-40;
 @block__border-top:             @gray-40;
+@block__border-right:           @gray-40;
 @block__border-bottom:          @gray-40;
-@block__bg:                     @gray-10;
+@block__border-left:            @gray-40;
```

Comment:
Same comment as the other PR, might as well point each side to the main `@block_border` var and let users update them individually only if needed.